test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -4157,12 +4215,7 @@ static void traffic_mngr_test_thresholds(void)
 
 static int traffic_mngr_check_queue_stats(void)
 {
-	odp_tm_capabilities_t capa;
-
-	if (odp_tm_capabilities(&capa, 1) < 1)
-		return ODP_TEST_INACTIVE;
-
-	if (capa.queue_stats.all_counters == 0)
+	if (tm_capabilities.queue_stats.all_counters == 0)
```

Comment:
Rebased and modified this to use tm_capabilities as odp_tm_capabilities() is deprecated